src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -448,6 +454,8 @@ public void stopsim() {
   
   @Override
   public void periodic() {
+    SmartDashboard.putData(this);
+    SmartDashboard.putBoolean("Safety", getChooChooAngle() > kMinChooChooAngle && getChooChooAngle() < kMaxChooChooAngle);
```

Comment:
Was this added in a merge?  Looks like it might be debug code, and maybe shouldn't stay

Reply:
it was, i've removed it.